apps/test/unit/gamelab/spritelab/locationCommandsTest.js

```diff
@@ -0,0 +1,44 @@
+import {expect} from '../../../util/configuredChai';
```

Comment:
use `../../../util/reconfiguredChai` as configuredChai contains a module that is being deprecated.